asciidoctor-stylesheet/sass/pages/docs.scss

```diff
@@ -6,6 +6,7 @@ body.book.toc2.toc-left {
   }
 
   #header {
+    width: 100% !important;
```

Comment:
Why do you need this?

Reply:
Because I want to get the height of the parent element of #header, but I think it is not necessary now.